packages/domain/candidature/src/corriger/corrigerCandidature.behavior.ts

```diff
@@ -116,9 +130,13 @@ export function applyCandidatureCorrigée(
 ) {
   this.importé = true;
   this.statut = StatutCandidature.convertirEnValueType(payload.statut);
-  this.typeGf =
-    payload.typeGarantiesFinancières &&
-    TypeGarantiesFinancières.convertirEnValueType(payload.typeGarantiesFinancières);
+  this.garantiesFinancières = payload.typeGarantiesFinancières
+    ? {
+        type: TypeGarantiesFinancières.convertirEnValueType(payload.typeGarantiesFinancières),
+        dateEchéance:
+          payload.dateÉchéanceGf && DateTime.convertirEnValueType(payload.dateÉchéanceGf),
+      }
+    : this.garantiesFinancières;
```

Comment:
Y a un intérêt à retourner le meme etat des garantiesFinancières si pas de typeGF dans le payload ?


```suggestion
  if (payload.typeGarantiesFinancières) {
    this.garantiesFinancières = {
      type: TypeGarantiesFinancières.convertirEnValueType(payload.typeGarantiesFinancières),
      dateEchéance:
        payload.dateÉchéanceGf && DateTime.convertirEnValueType(payload.dateÉchéanceGf),
    }
  }
```

Reply:
je vois que ce point est résolu mais ca me parait effectivement toujours à propos, non ?

Reply:
Dans correction tout est overriden donc j'ai ré-aligné le fonctionnement